models/modelsmodule.cpp

```diff
@@ -432,48 +366,12 @@ ModelsModule::init( SLIInterpreter* )
     "music_message_in_proxy" );
 #endif
 
-  // register synapses
-
-  /* BeginDocumentation
-     Name: static_synapse_hpc - Variant of static_synapse with low memory
```

Comment:
Where does this documentation go?

Reply:
@heplesser: the documentation is now [here](https://github.com/nest/nest-simulator/pull/1069/files#diff-4d73c5bddaa99f5c5da31c447c96dc33R185).